integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -70,16 +75,32 @@ def __init__(self, user, enterprise_configuration):
         """
         super(ContentMetadataExporter, self).__init__(user, enterprise_configuration)
         self.enterprise_api = EnterpriseApiClient(self.user)
+        self.enterprise_catalog_api = EnterpriseCatalogApiClient(self.user)
 
     def export(self, **kwargs):
         """
         Return the exported and transformed content metadata as a dictionary.
         """
         content_metadata_export = {}
-        content_metadata_items = self.enterprise_api.get_content_metadata(
-            self.enterprise_customer,
-            enterprise_catalogs=self.enterprise_configuration.customer_catalogs_to_transmit
-        )
+        request = crum.get_current_request()
+        if request and waffle.flag_is_active(request, USE_ENTERPRISE_CATALOG):
+            LOGGER.info(
+                'ENT-2572 [1] - Getting catalog metadata for enterprise [%s] from Enterprise Catalog Service.',
+                self.enterprise_customer.name
+            )
+            content_metadata_items = self.enterprise_catalog_api.get_content_metadata(
```

Comment:
i think it will be important to log the results of what we're getting back from this call for testing purposes, as well as which catalogs we are requesting data for.

Reply:
Logging string updated ✅